src/Http/Controllers/InvitationsController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Controllers;
+
+use Blumilk\Meetup\Core\Events\SendInvitationEmailEvent;
+use Blumilk\Meetup\Core\Http\Requests\Invitations\StoreInvitationRequest;
+use Blumilk\Meetup\Core\Services\InvitationsService;
+use Illuminate\Contracts\Auth\Guard;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+
+class InvitationsController extends Controller
+{
+    public function create(): View
+    {
+        return view("invitation");
+    }
+
+    public function store(StoreInvitationRequest $request, InvitationsService $service, Guard $auth): RedirectResponse
+    {
+        if ($service->existed($request->validated("email"))){
```

Comment:
```suggestion
        if (User::query()->where("email", $request->validated("email"))->exists()){
```
u can use query like that instead of `$service` which has one method.